verbose.c

```diff
@@ -232,7 +233,8 @@ char *token_to_string(Token *tok)
     case TTYPE_STRING:
         string_appendf(s, "\"%s\"", get_strtok(tok));
         return get_cstring(s);
+    default:
+        error("internal error: unknown token type: %d", get_ttype(tok));
```

Comment:
Move this statement out of `switch` scope.

Reply:
But if I mover this statement out of switch, the output by "make check": 
verbose.c:219:5: error: enumeration value ‘TTYPE_NULL’ not handled in switch [-Werror=switch]
     switch (ttype) {
     ^~~~~~

Reply:
I misinterpreted the context.